.gitignore

```diff
@@ -115,7 +115,7 @@ dist
 /reports
 
 # production
-
+build/
```

Comment:
Do we want to change th out to dist to align with csr?

Reply:
Done